Fortinet/fortiproxy/_meta/fields.yml

```diff
@@ -0,0 +1,337 @@
+  type: keyword
+fortinet.lanout:
+  description: 'LAN outgoing traffic in bytes'
+  name: fortinet.lanout
+  type: keyword
+fortinet.app:
+  description: 'Name of the application'
+  name: fortinet.app
+  type: keyword
+fortinet.apprisk:
+  description: 'Risk level of the application'
+  name: fortinet.apprisk
+  type: keyword
+fortinet.group:
+  description: 'User Group Name'
+  name: fortinet.group
+  type: keyword
+fortinet.reqtype:
+  description: 'Request type'
+  name: fortinet.reqtype
+  type: keyword
+fortinet.filteridx:
+  description: 'DLP filter ID'
+  name: fortinet.filteridx
+  type: keyword
+fortinet.filtercat:
+  description: 'DLP filter category'
+  name: fortinet.filtercat
+  type: keyword
+fortinet.dlpextra:
+  description: 'DLP extra information'
+  name: fortinet.dlpextra
+  short: keyword
+  type: keyword
+fortinet.profile:
+  description: 'DLP profile name'
+  name: fortinet.profile
+  type: keyword
+fortinet.filtertype:
+  description: 'DLP filter type'
+  name: fortinet.filtertype
+  type: keyword
+fortinet.eventid:
+  description: 'The serial number of the dlparchive file in the same epoch'
+  name: fortinet.eventid
+  type: keyword
+fortinet.severity:
+  description: 'Severity level of a DLP rule'
+  name: fortinet.severity
+  type: keyword
+fortinet.resptype:
+  description: 'Request type'
+  name: fortinet.resptype
+  type: keyword
+fortinet.reqtime:
+  description: 'Request time'
+  name: fortinet.reqtime
+  type: keyword
+fortinet.resptime:
+  description: 'Response time'
+  name: fortinet.resptime
+  type: keyword
+fortinet.respfinishtime:
+  description: 'Response finish time'
+  name: fortinet.respfinishtime
+  type: keyword
+fortinet.incidentserialno:
+  description: 'Incident serial number'
+  name: fortinet.incidentserialno
+  type: keyword
+fortinet.proto:
+  description: 'Integer of protocol used by web traffic'
+  name: fortinet.proto
+  type: keyword
+fortinet.filtername:
+  description: 'DLP rule name'
+  name: fortinet.filtername
+  type: keyword
+fortinet.fctuid:
+  description: 'fct uid'
+  name: fortinet.fctuid
+  type: keyword
+fortinet.forwardedfor:
+  description: 'forwardedfor'
+  name: fortinet.forwardedfor
+  type: keyword
+fortinet.sender:
+  description: 'Email addresses from the SMTP envelope'
+  name: fortinet.sender
+  type: keyword
+fortinet.subject:
+  description: 'The subject title of the email message'
+  name: fortinet.subject
+  type: keyword
+fortinet.recipient:
+  description: 'Email addresses from the SMTP envelope'
+  name: fortinet.recipient
+  type: keyword
+fortinet.trueclntip:
+  description: 'Trueclntip'
+  name: fortinet.trueclntip
+  type: keyword
+fortinet.unauthuser:
+  description: 'unauthuser'
+  name: fortinet.unauthuser
+  type: keyword
+fortinet.unauthusersource:
+  description: 'unauthusersource'
+  name: fortinet.unauthusersource
+  type: keyword
+fortinet.vrf:
+  description: 'VRF'
+  name: fortinet.vrf
+  type: keyword
+fortinet.direction:
```

Comment:
Can't we use the builtin `network.direction` field ? A mapping will be necessary to get the appropriate value.

https://www.elastic.co/guide/en/ecs/current/ecs-network.html#field-network-direction

Reply:
Nice, in this case it is OK for me to keep the original value in a custom field ;)